playwright/README.md

```diff
@@ -9,23 +9,28 @@
 
 ## Installation
 
-We are planning to use [Playwright](https://playwright.dev) testing framework for functional and regression testing. Playwright is already installed in the Carbon project. Clone and install the carbon repository to apply the installation:
+Ensure Carbon is first installed on your machine by doing:
```

Comment:
I think it would read better if this ended with "the following:", i.e. "Ensure Carbon is first installed on your machine by doing the following:".